.github/workflows/build-test-manylinux-aarch64.yaml

```diff
@@ -3,6 +3,13 @@ name: build-test-manylinux-aarch64
 on:
   workflow_dispatch:
   workflow_call:
+    outputs:
```

Comment:
Honestly the way we do things in Yaml looks completely foreign to me. If we were in a sane programming language,
we could just return values, takes parameters, and we would not need all those weird way to do simple things.